wiki/osu!_Program_Files/zh.md

```diff
@@ -16,32 +16,32 @@ osu! 目录文件
 文件夹
 ------
 
-### `_MACOSX`
+### _MACOSX
```

Comment:
Don't know why there's an underscore. Update on `en.md` too. https://en.wikipedia.org/wiki/MacOS
```suggestion
### macOS
```

Reply:
It's styled this way because that's how the directory is named, each header lists a directory

Reply:
Can you apply the backslash, edited the suggested.